### ISSUE TITLE
Is Armchair still being maintained?

### ISSUE THREAD

**Issue by freak4pc** [UrbanApps/Armchair#84]
It seems the last commit is from 3 months ago and the master branch is still using Swift 2.3. 

Is this repo still actively maintained ?

**Comment by coneybeare** [comment]
Yes. While I have been taking the back burner lately, pull requests are still checked and merged if appropriate. I will merge in the swift 3 branch soon
